cmd/bridge/main.go

```diff
@@ -88,6 +88,7 @@ func main() {
 	fK8sAuth := fs.String("k8s-auth", "service-account", "service-account | bearer-token | oidc | openshift")
 	fK8sAuthBearerToken := fs.String("k8s-auth-bearer-token", "", "Authorization token to send with proxied Kubernetes API requests.")
 
+	fCustomHostnameRedirect := fs.Bool("custom-hostname-redirect", false, "When true, console backend will listen on port 8080 to redirect to the base address.")
```

Comment:
This should be the port number rather than a boolean. Something like `redirect-port`. Only redirect if set.

Reply:
Though that we wanna hardcode the port number.

Reply:
No, even if only for local development, it should be configurable to avoid port conflicts. The operator can hard-code a value, though.

Reply:
Something like 8444 is a better default btw since it's https.